Enhance README.md with updated payment methods and integration notes.… (#5)
zeptonow/mcp: README.md

After authentication, the MCP can securely access your Zepto account.
Authentication can be cleared directly from the client if needed.

### Allowed Redirect URIs

The following redirect URIs are whitelisted for MCP authentication:

-   `claude://claude.ai/settings/connectors`
-   `https://claude.ai/api/mcp/auth_callback`
-   `https://insiders.vscode.dev/redirect`
-   `https://oauth.pstmn.io/v1/callback`
-   `https://vscode.dev/redirect`
-   `http://localhost`
-   `http://localhost/callback`
-   `http://127.0.0.1`
-   `http://127.0.0.1/callback`

> Contact us if you need additional URIs whitelisted.

Reviewer: ```suggestion
Please raise an issue you need additional URIs whitelisted.
```

### 2. Reorder From History

> "What did I order last time? Reorder the same items."

The AI retrieves your most recent order, lists out the items, and adds
them back to your cart so you can place the order again with a single
confirmation.

### 3. Recipe-Based Shopping

> "I want to make paneer butter masala tonight for 4 people. Figure out
> what ingredients I need and order them from Zepto."

The AI generates a recipe ingredient list, searches for each item on
Zepto, picks the right quantities, adds everything to your cart, and
gives you a total — turning a dinner idea into a ready-to-place order.

### 4. Order From a Photo

> _\[Attach a photo of a recipe card, a screenshot of a dish, or a
> fridge with missing items\]_
>
> "Here's a recipe I found — order everything I'd need to make this."

The AI reads the image, identifies the ingredients or items, searches
for them on Zepto, and builds your cart. Works with recipe screenshots,
handwritten lists, or even a photo of a dish you want to recreate.

### 5. Occasion-Based Shopping

> "I'm hosting a Diwali party for 15 people this weekend. I'll need
> sweets, dry fruits, snacks, drinks, and some basic decorations if

> > available. Put together a cart for me."

Reviewer: ```suggestion
> "I'm hosting a Diwali party for 15 people this weekend. I'll need
> sweets, dry fruits, snacks, drinks, and some decorations. Put together a cart for me."
```
